### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tiberiu44**
# Contribution Guidelines

Please ensure your pull request adheres to the following guidelines:

- Search previous suggestions before making a new one, as yours may be a duplicate.
- Make an individual pull request for each suggestion.
- Use the following format: [Bookmark Title](link): Description.
- The title usually consists of several words. For bookmarks of an article or tutorial, use the title of the article.
- The description consists of one-two sentences about the bookmark. It is required.
- If the resource has 2 or more links please provide the Github repo link.
- Additions should be added to the bottom of the relevant category.
- New categories or improvements to the existing categorization are welcome. 
- Check your spelling and grammar.
- Make sure your text editor is set to remove trailing whitespace.
- The pull request and commit should have a useful title.

Thank you for your suggestions!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,6 +381,12 @@ Memory networks are implemented in [MemNN](https://github.com/facebook/MemNN).
 
 * [DeepNLP-models-Pytorch](https://github.com/DSKSD/DeepNLP-models-Pytorch) has Pytorch implementations of various deep NLP models used in CS224n(Stanford) in the form of Jupyter notebooks.The models are aimed for those who are acquainted with Pytorch.
 
+## Multilingual NLP Frameworks
+
+[Back to Top](#contents)
+* [UDPipe](https://github.com/ufal/udpipe) : Trainable pipeline for tokenizing, tagging, lemmatizing and parsing Universal Treebanks and other CoNLL-U file
```
Comment (NirantK) [inline README.md]: Hey @tiberiu44 , thanks for the PR!

Have you used UDPipe personally? Can you vouch for it's performance and stability?

Comment (tiberiu44) [reply README.md]: You are right. I'll update the two descriptions and include Spacy after I get the review from my friend. Thanks.

Comment (NirantK) [reply README.md]: Thanks @tiberiu44 

We have Spacy in the README. If you'd like to add a specific comment/review of why you like it - we'd like that :)

**Comment by Dmitriy11111** [comment]
bc733d2

**Pull request merged**